proto/p4/p4runtime.proto

```diff
@@ -136,6 +136,14 @@ message TableEntry {
   // this is stored, as long as it is returned with the rest of the entry in
   // a Read RPC.
   uint64 controller_metadata = 5;
+  // meter_config and counter_data are convenience fields that enable the
+  // controller to configure the direct resources associated with the table at
+  // the same time as a match-action entry is inserted or modified. When
+  // updating a table entry, leaving these fields unset means that the direct
+  // resources (if any) will not be updated. When reading a table entry, the
+  // values associated with the direct resources (if any) will be returned.
```

Comment:
As I interpret this, I see a conflict between what this comment says and what Yavuz is expecting in the draft runtime spec.  The runtime spec says that the reply to a Read must equal the original Write request.  This comment states that the direct resources should be included in a Read, with no dependency on whether they were part of the original write.  If the controller chooses not to include them in the Write, then the reply violates the runtime spec if the controller adds any direct resource later.